SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -273,14 +280,40 @@ extension ViewController: PushKitEventDelegate {
             } else {
                 NSLog("Successfully registered for VoIP push notifications.")
                 
+                // Save the device token after successfully registered.
+                UserDefaults.standard.set(cachedDeviceToken, forKey: kCachedDeviceToken)
+                
                 /*
-                 * Save the device token after successfully registered.
+                 * The twilio registration binding is refreshed for 365 days upon regirstrtion and when a new
+                 * push notification is dispatch.
+                 * Reset the binding date in UserDefaults.
                  */
-                UserDefaults.standard.set(cachedDeviceToken, forKey: kCachedDeviceToken)
+                UserDefaults.standard.set(Date(), forKey: kCachedBindingDate)
             }
         }
     }
     
+    /*
+     * The twilio registration binding is refreshed for 365 days on twilio notify when a new
+     * push-notification is disptached. This method checks if binding exists in UserDefaults,
+     * and if 365 days has been passed then the method will return true, else false.
+     */
+    func bindingRequired() -> Bool {
```

Comment:
Lets call this `registrationRequired()`. Exeternally we refer to this as registration so I would prefer to stick with that terminology.